### PR TITLE
InSAR Water Mask descriptions

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

File: hyp3_metadata/templates/insar/readme.md.txt.j2 [hunk 1]
```diff
@@ -177,6 +178,15 @@ The DEM used for this product is {{ dem_name }}, which has a native pixel spacin
 *Refer to the _dem.tif.xml file for additional information about the specific DEM included with this product, including use and citation requirements.*
 
 {% endif %}
+----------------
+## 11. Water Mask
+
+The water mask specifies, pixel-by-pixel, where large waterbodies exist in the corresponding InSAR product. Pixel values of 0 indicate areas that should be masked out, while areas with a pixel value of 1 are included in the InSAR processing. This mask is applied to the DEM prior to running InSAR jobs, effectively clipping out the water, as areas with DEM heights of 0 are considered invalid for InSAR processing. The water mask is stored as a byte-valued GeoTIFF file and is included in the product package when water masking is applied.
```
Comment (jhkennedy) [inline hyp3_metadata/templates/insar/readme.md.txt.j2]: > This mask is applied to the DEM prior to running InSAR jobs, effectively clipping out the water, as areas with DEM heights of 0 are considered invalid for InSAR processing.

I don't think this is how it works anymore, and will need to be updated

**Review by jhkennedy — COMMENTED** [review]
We'll probably also need an ArcGIS XML template?

**Comment by hjkristenson** [comment]
> We'll probably also need an ArcGIS XML template?

There's a user story for that if we want to bring it onto this sprint: 
https://asfdaac.atlassian.net/browse/APD-2102

**Comment by hjkristenson** [comment]
The other to-do for this is to remove the notification box in the Product Guide saying that the water mask functionality is coming soon.

**Review by hjkristenson — COMMENTED** [review]
This draft template was written assuming the layer will be included but that it will not be applied to processing. It will need to be updated if/when functionality is added to apply the mask to the InSAR products.

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**